docs/blitz/details/yang.rst

```diff
@@ -111,6 +111,7 @@ subscription, or, you may expect the test to fail (referred to as a negative tes
       auto-validate:   # [true | false] automatically validate config messages
       negative-test:   # [true | false] expecting device to return an error
       pause:           # pause N seconds between each test (default: 0, no pause)
+      sequence:        # [true | false] to check the return values and return sequence verified
```

Comment:
You can drop the "to check the" from the sentence.  What we really need is a detailed description of each option in format following this shorthand but I am not going to put that on you.

Reply:
updated.